### ISSUE TITLE
SC 211 -- Select box support for Enter/Return key MacOS

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#29]
## URL
https://demo.fileyourstatetaxes.org/en/questions/nj-county

## Issue
PC users can use the <kbd>Enter</kbd> key on the `select` element but Mac users cannot use the <kbd>Return</kbd> key. 

## Video walkthrough
https://github.com/user-attachments/assets/ef4e1e71-5307-4409-a057-49f28095cd14

## WCAG Success Criterion
[SC 2.1.1: Keyboard (Level A)](https://w3.org/TR/WCAG21#keyboard)

## Impacted users
- People who are blind (who cannot use devices such as mice that require eye-hand coordination)
- People with low vision (who may have trouble finding or tracking a pointer indicator on screen)
- Some people with hand tremors find using a mouse very difficult and therefore usually use a keyboard

## Suggested fix

Follow guidance from W3C captured here: 
https://codepen.io/dsian108/pen/VYZrJPW
